engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -193,31 +162,26 @@ public boolean pregenerateChunks() {
     }
 
     @Override
-    public void update(float delta) {
+    public void update(float deltaInMs) {
 
         PerformanceMonitor.startActivity("Update Tick");
```

Comment:
I'll rename the parameter appropriately. This prompts a question though: why does the animation system needs times in 1000s of seconds?

Thank you for the information about PerformanceMonitor, now I understand its purpose and usage much better. For what I have seen in the code there isn't a huge amount of multiple return location. But I didn't think about this aspect before and I'll now double-check to make sure that each startActivity is appropriately matched by an endActivity. Given that this is the only concern on the matter however, I will generally move those statements within methods. I really feel that they fragment the code too much, especially the high-level methods. And the issue of multiple return location is probably easily addressable within this area of the codebase.